BEST_PRACTICES.md

```diff
@@ -253,6 +254,18 @@ Views should be layout using Apple's Auto Layout. No third-party frameworks are
 
 Old style layout is still an option for when Auto Layout is not available.
 
+## Optional Force Unwrapping
+
+When something that shouldn't return an optional, returns an optional you have two options:
+
+**A)** Pass the error to the caller (throw): This is the ideal way of handling it, specially if there's the possibility of the app to recover. It's pretty straight forward, you create an `Error` or `NSError` with some information about the error, and throw it. [More information about throwing here.](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/ErrorHandling.html)
+
+For example, lets say you have a method that returns a filename for a photo, in theory all the photos should have a filename, so, the return value shouldn't be an optional. Throw an error in this case and let the method caller decide if they either go for crashing the app, or for just excluding the photo with the missing filename.
```

Comment:
"for example" and "lets say…" both have the same function. We use one or the other. 👍